modules/cassandra/src/test/java/org/testcontainers/containers/CassandraServer4Test.java

```diff
@@ -0,0 +1,35 @@
+package org.testcontainers.containers;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CassandraDriver4Test {
```

Comment:
filename and class name doesn't match.
```suggestion
public class CassandraServer4Test {
```